docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: PATCH updating storage
+---
+
+`PATCH /asset/:id -d { }` can be used to modify any `asset` that was uploaded with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
+
+
+## Update Stored Asset
+
+
+## Updating Asset Name
+
+This will allow the name of the stored asset in Livepeer Studio to be changed.
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d { "name": "New name" }
+```
+
+## Updating Storage
+
+This allows to add additional storage location of where the assets are to be stored.
+
+>**NOTE:** 
+- Currently assets cannot be deleted from the default Livepeer Studio storage. You can  customize any additional storage though. 
+- Only IPFS is supported as an additional storge at the moment. We are currently working on support for other decentralized storages like Arweave and Storj.
+
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {"storage": {"ipfs": {}}}
+```
+
+## Updating Metadata
+
+This allows for modifying the metadata of the asset, which is customizable NFT. Since each marketplace has their own standard for the `metatdata` of the NFT, it is up to you to determine which standard to use and fields to include.
+
+>**NOTE:** 
+- Livepeer Studio's defaut fields for `metadata` are:
+```json
+    "name": ,
+    "description": ,
+    "cid": 
```

Comment:
Ops, had sent a comment on the commit by mistake. Sending it again on the PR here.

This is not the right field name, it is actually `animation_url` and `properties.video`, following ERC-1155.

You should also add some instructions on how to override/delete these default fields, as the curr CLi print as well:
https://github.com/livepeer/video-nft/blob/main/src/cli/args.ts#L123-L126

Can also use the SDK docs for some further reference: https://livepeer.github.io/video-nft/classes/minter.Api.html#exportToIPFS

And finally, can resort to the implementation here to see the (default) metadata in detail, which can be overridden by what is set here: https://github.com/livepeer/task-runner/blob/4d1ae1768c01e979969bf7f00b81267db05e299c/task/export.go#L155

Let me know if you still need any additional info. I know that these are not very structured pieces of documentation so I can help clarify things further.

Reply:
Also, upon revisiting the last link I sent with the documentation, I noticed we're populating a few other fields by default. The extensive list with some short description (iow, my take on this doc):

> - `name`: Populated with the asset name
> - `description`: A default description referring to the asset name above
> - `image`: A default image including the Livepeer logo as a thumbnail
> - `animation_url`: URL for the Livepeer player on IPFS playing the video from Livepeer Studio
> - `external_url`: URL for the Livepeer player hosted on lvpr.tv playing the video from Livepeer Studio
> - `properties["video"`: CID of the original video file saved on IPFS
> - `properties["com.livepeer.playbackId"]`: playback ID of the asset in Livepeer Studio
>
> Notice that you can change the value or delete any of these default fields by providing a value in the `nftMetadata` field. You can delete any of the above fields by specifying them as `null` in your overrides.